.github/workflows/daily.yml

```
@@ -876,6 +876,9 @@ jobs:
         run: dnf -y install epel-release
       - name: make
         run: |
+          if [[ "${{ matrix.name }}" == "test-fedoralatest-jemalloc" ]]; then
+            dnf -y install awk
+          fi
           dnf -y install gcc make procps-ng which /usr/bin/kill
```

Can we just add awk to the list of packages to install? In my "not latest" fedore, the package is called `gawk` but it should also work to specify the path of the binary you want, like `/usr/bin/awk`.

```suggestion
          dnf -y install gcc make procps-ng which /usr/bin/kill /usr/bin/awk
```

If the above doesn't work, then can we can check if awk exists instead of matching the hard-coded name of the CI job, such as like below?

```
          which awk >/dev/null || dnf -y install awk
```

---

I tested on Fedora 41 and 42 using Docker. Fedora 41 has awk, but Fedora 42 has neither awk nor gawk. I think this is why the CI is failing now, as the latest Fedora has removed awk, and we need to manually install it.

Yeah, you’re absolutely right — the command `which awk >/dev/null || dnf -y install awk` is indeed more user-friendly.

---

I was talking about the package name. On Fedora 40 there is no package `awk` but the package name is `gawk` and it provides `/usr/bin/awk` and `/bin/awk`. I was thinking that we can install `/usr/bin/awk` it would work on any version, and if it is already installed, it is a no-op.